tests/Admin/PoolTest.php

```diff
@@ -376,6 +388,141 @@ public function getInvalidChildAdminServiceNames()
         ];
     }
 
+    /**
+     * @dataProvider getAdminServiceNamesToCheck
+     */
+    public function testHasAdminByAdminCode(string $adminId): void
+    {
+        $adminMock = $this->getMockBuilder(AdminInterface::class)
```

Comment:
Please use createMock() instead

Reply:
Done. Also changed the ones introduced by @phansys for the `Pool::getAdminByAdminCode()` tests.

Reply:
Thanks for checking the tests. Hadn't had any feedback on them so far. If you see anything else in there that might need attention, please let me know